src/components/card/card-view.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react';
+import { GrChannel } from 'react-icons/gr';
+import { IconContext } from 'react-icons';
+import Header from '../common/Header';
+
+const CardView = (props) => {
+  const [video, setVideo] = useState(props.location.query?.video);
```

Comment:
I recommend using something like this: 
`const { videoId } = useParams()`
[Link to official documentation](https://reactrouter.com/web/api/Hooks/useparams)

This is for the 4th deliverable, so no worries 😉